### ISSUE TITLE
Updating the port on Macports for totp-cli (v1.2.5)

### ISSUE THREAD

**Issue by RobK88** [yitsushi/totp-cli#62]
The existing port for `totp-cli` (v1.1.17) on Macports is broken for the latest versions of macOS (but strangely enough the port works on old versions of macOS like Lion).

I have submitted a Pull Request to the Macports repo that fixes the port.  The Pull Request also updates`totp-cli` to the latest version (v1.2.5).  

See https://github.com/macports/macports-ports/pull/17418

The updated port appears to work just fine.  It builds and runs `totp-cli` v1.2.5 on my old Mac running Lion.  It also builds and runs just fine on Big Sur and Monterey and presumably runs just fine on Ventura too!

I needed to add a missing dependency (golang/x/term) and change one of the dependencies (golang.org/x/sys) to the latest version!

(P.S.  The reason why `totp-cli` builds and runs just fine on Lion is due to the Macports Legacy support library.  So users trying to build `totp-cli` on old Macs should use Macports and not try to build directly using Go).

I am not sure whether any of the other dependencies listed in the Portfile need to be updated for bug fix or security reasons.  I will take a look.

It would be appreciated if you could review the attached Portfile.  If you see anything that should be updated (like a dependency), please let me know.

Thanks,

RobK88
[Portfile.txt](https://github.com/yitsushi/totp-cli/files/10473442/Portfile.txt)

**Comment by RobK88** [comment]
I have updated all of the dependencies in the Portfile for `totp-cli` v1.2.5.  (see Attached) and have pushed the changes to the Pull Request on Macports.

There is only one dependency that I am not sure of.  It is a dependency written by you -- `github.com/yitsushi/github-release-check`.  The Portfile has it set (or locked) at v1.0.1.  I do not even see that tag on GitHub.  But Macports is still able to pull the code when the port is being installed. 

Is R1.0.1 the latest release for `github-release-check`?  Should this dependency be updated?  Please let me know.  

Thanks!
[Portfile Updated.txt](https://github.com/yitsushi/totp-cli/files/10474708/Portfile.Updated.txt)

**Comment by yitsushi** [comment]
Note: I'm not the maintainer of the macports package, and I don't really have any work or connection with that. I'm happy it exists, but that's it.

For the release check, test `1.0.1` is the latest
![image](https://user-images.githubusercontent.com/183191/213990286-d8fe65e0-3dd5-430a-bff4-7d13801e15ae.png)

**Comment by RobK88** [comment]
Thanks!  I should have checked the tags listed on your website for `github-release-check`.  Instead I looked at releases (on the right hand side of the web page)  which only shows release 1.0.

I am a contributor and maintainer for a number of ports at MacPorts.  I noticed that the port for `totp-cli` was out of date and would not even install on recent versions of macOS (High Sierra) and later.  So I took the initiative and fixed the port.  I have submitted a Pull Request (PR).  It now builds fine of old releases of macOS (like Lion) and the most recent releases of macOS (like Big Sur and Monterey).  The PR should be reviewed and merged in a week or so.

Thanks again for developing `totp-cli`!
